internal/svcctl/comm.go

```diff
@@ -57,7 +63,7 @@ func (c *Comm) GetLogFileName(ctx context.Context) (string, error) {
 }
 
 type RuntimeUsageReporter interface {
-	ReportRuntimeUsage(ctx context.Context, pid, exec, namespace, commit, headless string) error
+	RuntimeUsage(ctx context.Context, pid int, exec, dimensionsJSON string) (*graph.RuntimeUsageResponse, error)
```

Comment:
Huh, it looks like this was always passed as an int:

https://github.com/ActiveState/cli/blob/22be2a51eb0bca4c8c80e4a103a0b72ee1dad738/cmd/state-svc/internal/server/generated/generated.go#L1009

How was this working before?

Reply:
I'm not following why it wouldn't work before.

Reply:
Cause pid on `RuntimeUsage` was a string, but it was passing an int.